.github/workflows/pages.yml

```diff
@@ -0,0 +1,50 @@
+name: Deploy manual
+
+on:
+  push:
+    branches: [master]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
```

Comment:
i don't know what this actually does. But do we need to write on any token? Or is it updated on any run?

Reply:
https://github.com/actions/deploy-pages
https://github.com/actions/deploy-pages/issues/329

I've left the comments in now.